packages/webapp/src/pages/play.tsx

```diff
@@ -215,41 +213,45 @@ const Play: FablePage = ({ isHydrated }) => {
 
             {!ended && (
               <>
-                <button
-                  className="btn-warning btn-lg btn absolute right-96 bottom-1/2 z-50 !translate-y-1/2 rounded-lg border-[.1rem] border-base-300 font-mono hover:scale-105"
+                <Button
+                  variant={"secondary"}
+                  className="absolute right-96 bottom-1/2 z-50 !translate-y-1/2 rounded-lg border-[.1rem] border-base-300 font-mono hover:scale-105"
                   disabled={cantDrawCard}
                   onClick={doDrawCard}
                 >
                   DRAW
-                </button>
+                </Button>
 
-                <button
-                  className="btn-warning btn-lg btn absolute right-48 bottom-1/2 z-50 !translate-y-1/2 rounded-lg border-[.1rem] border-base-300 font-mono hover:scale-105"
+                <Button
+                  variant={"secondary"}
+                  className="absolute right-48 bottom-1/2 z-50 !translate-y-1/2 rounded-lg border-[.1rem] border-base-300 font-mono hover:scale-105"
                   disabled={cantEndTurn}
                   onClick={doEndTurn}
                 >
                   END TURN
-                </button>
+                </Button>
```

Comment:
These buttons don't look great when disabled (with the transparency that shows the line behind them).

Reply:
We can customise this as we want to within the component file itself, since it's its own script and not predefined styles within the package.

Reply:
True, anyhow we probably should do a detailed UX pass before the PoC to tweak all the small stuff like this.